webpack/ForemanTasks/Components/common/ActionButton/ActionButton.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { Button } from 'patternfly-react';
+import PropTypes from 'prop-types';
+import { translate as __ } from 'foremanReact/common/I18n';
+
+export const CancelButton = ({ id, name, cancellable, cancel }) => (
```

Comment:
IMO and based on the foreman convention `CancelButton` and `ResumeButton` should live in their own files.